### DISCUSSION TITLE
Get custom node name in a workflow

### DISCUSSION THREAD

**Discussion by abaybektursun** [Comfy-Org/ComfyUI#3675] [Q&A]
This seems impossible right now. How can I extract the name or GitHub URL of a node on a workflow?
The ComfyUI has that info somewhere since you can install missing nodes via the manager.

**Comment by ltdrdata** [comment]
You can utilize this.
I am maintaining this mapping files.

**Reply by abaybektursun** [reply]
What is `this`? Did you mean to add a link?

**Reply by ltdrdata** [reply]
> What is `this`? Did you mean to add a link?

Oh... I had forgot pasting link. lol

https://github.com/ltdrdata/ComfyUI-Manager/blob/main/extension-node-map.json
